src/traffic participants/Vehicle.ts

```diff
@@ -0,0 +1,28 @@
+class Vehicle implements RoadUser {
+    velocity: number;
+    direction: Direction;
+    isRoadFree: boolean;
+    priority: Priority;
+
+    constructor(vel: number, dir: Direction) {
+        this.velocity = vel;
+        this.direction = dir;
+        this.priority = Priority.VEHICLE;
+    }
+    
+    go(): void {
+        if(this.velocity == 0) {
```

Comment:
In TS and JS, it's better to use the '===' equality operator rather than '=='. It is used to prevent type casting. It's also about '!==' operator